### PR TITLE
Fix a lot of checkstyle, FindBug and PMD violations

### PR DESCRIPTION
**Pull request by ghyl**
Still some left, probably will have to be done for the next iteration.

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/semgr8/model/PieceGroup.java [hunk 1]
```diff
@@ -50,6 +54,7 @@ public boolean add(Piece piece) throws RuntimeException {
      *
      * @return the score value
      */
+    @SuppressWarnings("checkstyle:magicnumber")
```
Comment (kiliangrashoff) [inline src/main/java/nl/tudelft/ti2206/semgr8/model/PieceGroup.java]: Add reason for warning

File: src/test/java/nl/tudelft/ti2206/semgr8/model/PieceGroupGetPowerUpTest.java [hunk 2]
```diff
@@ -14,6 +14,7 @@
 /**
  * Tests the getPowerUp method in PieceGroup.
  */
+@SuppressWarnings("checkstyle:magicnumber")
```
Comment (kiliangrashoff) [inline src/test/java/nl/tudelft/ti2206/semgr8/model/PieceGroupGetPowerUpTest.java]: Add reason

**Comment by kiliangrashoff** [comment]
LGTM, just need reasons for warnings supression in two places. Not really anything else to say, since you can't really argue with pmd and checkstyle.

**Comment by ghyl** [comment]
comments processed, will merge now

**Pull request merged**